tracer/src/Datadog.Trace/Debugger/Snapshots/SupportedTypesService.cs

```diff
@@ -65,11 +65,11 @@ internal static class SupportedTypesService
             typeof(SecureString),
         };
 
-        internal static bool IsSafeToCallToString(Type type)
+        internal static bool IsSafeToCallToString(Type type, bool includeCollection = true)
```

Comment:
This change seems like it might be adding needless complexity just to save one consumer some trouble - can we remove this boolean and instead have the consumer check if it's a collection or not?